sql/comments_database_setup.sql

```diff
@@ -123,6 +123,24 @@ CREATE TABLE StudyBuddyConnections (
     PRIMARY KEY (senderId, receiverId, courseId)
 );
 
+create TABLE AccessControl (
+id INT UNSIGNED NOT NULL AUTO_INCREMENT,
+description TEXT,
+updaterId INT UNSIGNED NULL,
+isOpen BOOLEAN,
+createdAt DATETIME,
+updatedAt DATETIME,
+PRIMARY KEY (id)
+);
+
+CREATE TABLE AccessControlMember(
+id INT UNSIGNED NOT NULL AUTO_INCREMENT,
```

Comment:
I tested this and got this error, `All pats of a PRIMARY KEY must be NOT NULL; if you need NULL in a key, use UNIQUE instead`. We should be able to put null in `accessControlMember` and `userMember` because one of the `userMember` or `accessControlMember` is null in any scenario. So I end up with id.

Reply:
Okay. Thsnks for checking. Let's keep it as is .